### PR TITLE
Changes in Lyrics and Thumbnails

### PR DESCRIPTION
**Pull request by aneesh1122**
1) Decreased sensitivity of horizontal swipe

https://github.com/fast4x/RiMusic/assets/45353488/b48dbfe9-553d-4115-9b9b-da95ce3443e0

2) added accent color to play/pause for essential disabled button
Screenshot_2024-06-15-15-23-36-666_it fast4x rimusic debug-edit
3) blur can go as low as 0f now
Screenshot_2024-06-15-15-24-35-414_it fast4x rimusic debug-edit
4) added options to hide thumbnail and hide  thumbnail behind lyrics
Screenshot_2024-06-15-15-25-57-114_it fast4x rimusic debug-edit

https://github.com/fast4x/RiMusic/assets/45353488/7ad974d8-f4c7-4e37-8231-82d6a76a7b57

5) added accent color to lyrics when "Show thumbnail" is disabled. Good for 0 blur background

https://github.com/fast4x/RiMusic/assets/45353488/52afa99b-bf79-43b4-9f56-e02025a72352

6) changes lyrics color according to the theme mode

https://github.com/fast4x/RiMusic/assets/45353488/e46161e8-1bb7-4560-8746-c944ab06b078

7)no problem of visibility. Added small shadow behind texts to increase visibility.
Screenshot_2024-06-15-15-46-40-620_it fast4x rimusic debug

8) changed modern disabled button a little bit. I'm working on removing that square shadow which I'll replace with shadows according to the shape.
Screenshot_2024-06-15-15-50-31-057_it fast4x rimusic debug

9) This is how full screen lyrics look like when thumbnail is disabled. also added an increased lyrics size

https://github.com/fast4x/RiMusic/assets/45353488/2c14d9a8-95e6-48c1-9ade-c6dfce7f3910

Note : I couldn't find a way to add stroke properly so decided to add shadows. I'll be searching for a way to add the stroke/text outline.

Lyrics with accent color works only when 'show thumbnail' is deactivated.

### CODE REVIEW

**Comment by fast4x** [comment]
Damn how many changes you made, can you assure me that before you changed the code you updated it with release 39?

**Comment by aneesh1122** [comment]
> Damn how many changes you made, can you assure me that before you changed the code you updated it with release 39?
> 

Yeah. Funny story.

I developed these things yesterday and since I know little about GitHub I foolishly pushed the files. It was a mess

So I deleted my previous repo, added the new codes again and cloned the repo again today and then pushed the new changes.

TLDR; yeah, These changes are over your latest update.

**Comment by aneesh1122** [comment]
> Damn how many changes you made, can you assure me that before you changed the code you updated it with release 39?
> 

I wanted to change something but I want to ask you before doing that.

Right now accent color lyrics are activated only when "show thumbnails" is disabled.

Should I keep accent color lyrics even when "show thumbnails" is enabled?

**Comment by fast4x** [comment]
No, for now it's ok.

**Comment by fast4x** [comment]
Merged, available from version 0.6.40
when create new translation, use lamp suggestione in Android Studio, click on string, click on yellow lamp and then click extract resource string...
In this mode, all languages are created...

**Comment by aneesh1122** [comment]
> Merged, available from version 0.6.40
> when create new translation, use lamp suggestion in Android Studio, click on string, click on yellow lamp and then click extract resource string...
> In this mode, all languages are created...

Thanks man. I'm new to GitHub and Android studio so I still have a lot to learn. I'll make sure to do what you told me the next time.

Btw I've added stroke to lyrics text. Should I create a new pull request?

**Comment by fast4x** [comment]
Yes Always create new pull request
